Resources/translations/SonataMediaBundle.de.xliff

```diff
@@ -567,6 +567,18 @@
                 <source>form.label_class</source>
                 <target>CSS Klasse</target>
             </trans-unit>
+            <trans-unit id="media_">
+                <source>Media</source>
+                <target>Medien</target>
+            </trans-unit>
+            <trans-unit id="gallery_">
+                <source>Gallery</source>
+                <target>Galerie</target>
+            </trans-unit>
+            <trans-unit id="options">
+                <source>Options</source>
```

Comment:
if we don't use keys there, we should do this in the next major IMO, what do you think @core23 ?